autogen/logger/logger_factory.py

```diff
@@ -2,6 +2,7 @@
 
 from autogen.logger.base_logger import BaseLogger
 from autogen.logger.sqlite_logger import SqliteLogger
+from autogen.logger.cosmos_db_logger import CosmosDBLogger
```

Comment:
As mentioned in #2329, we should do this in a try-except block to check for import error as cosmos db is going to be an optional dependency.